app/user/tests/test_user_api.py

```diff
@@ -0,0 +1,426 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+
+from user.tests.utils import (
+                        TestBase,
+                        TestBaseWithEmail,
+                        TestBaseLogin,
+                        TestBaseLogout,
+                        TestBaseUserChange,
+                        TestBaseUserResetPasswordEmail,
+                        TestBaseEmailVerify,
+                        TestBaseUserResetPasswordVerification,
+                        TestBaseUserResetPasswordConfirm,
+                      )
+
+
+USER = get_user_model()
```

Comment:
i kno it's `cleanest` way possible -> using `get_user_model`
IN realith tho, no one does that most of the time
just roll with `users.User` -> import User
More convenient